clustered/common/src/main/java/org/ehcache/clustered/common/messages/ResponseCodec.java

```diff
@@ -94,6 +96,15 @@
         buffer.putInt(((ClientInvalidateAll) response).getInvalidationId());
         return buffer.array();
       }
+      case SERVER_INVALIDATE_HASH: {
+        ServerInvalidateHash serverInvalidateHash = (ServerInvalidateHash) response;
+        byte[] cacheIdBytes = serverInvalidateHash.getCacheId().getBytes(Charset.forName("UTF-8"));
```

Comment:
I'm going to commonize the code that encodes strings, but the more I look at the message / codec code the more I think this should all be handled with something like Protobuf.